docs/content/SplitPageLayout.mdx

```diff
@@ -3,156 +3,211 @@ title: SplitPageLayout
+  <SplitPageLayout>
+    <SplitPageLayout.Header>
+      <Placeholder label="Header" height={64} />
+    </SplitPageLayout.Header>
+    <SplitPageLayout.Pane divider={{regular: 'line', narrow: 'none'}}>
+      <Placeholder label="Pane" height={120} />
+    </SplitPageLayout.Pane>
+    <SplitPageLayout.Content hidden={{narrow: true}}>
+      <Placeholder label="Content" height={320} />
+    </SplitPageLayout.Content>
+    <SplitPageLayout.Footer>
+      <Placeholder label="Footer" height={64} />
+    </SplitPageLayout.Footer>
+  </SplitPageLayout>
+</Box>
 ```
 
 ### Without dividers
 
-```jsx
-<SplitPageLayout>
-  <SplitPageLayout.Header divider="none">
-    <Placeholder label="Header" height={64} />
-  </SplitPageLayout.Header>
-  <SplitPageLayout.Pane divider="none">
-    <Placeholder label="Pane" height={120} />
-  </SplitPageLayout.Pane>
-  <SplitPageLayout.Content>
-    <Placeholder label="Content" height={240} />
-  </SplitPageLayout.Content>
-  <SplitPageLayout.Footer divider="none">
-    <Placeholder label="Footer" height={64} />
-  </SplitPageLayout.Footer>
-</SplitPageLayout>
+```jsx live drafts
+<Box sx={{height: 320, overflowY: 'auto', border: '1px solid', borderColor: 'border.default'}}>
+  <SplitPageLayout>
+    <SplitPageLayout.Header divider="none">
+      <Placeholder label="Header" height={64} />
+    </SplitPageLayout.Header>
+    <SplitPageLayout.Pane divider="none">
+      <Placeholder label="Pane" height={120} />
+    </SplitPageLayout.Pane>
+    <SplitPageLayout.Content>
+      <Placeholder label="Content" height={320} />
+    </SplitPageLayout.Content>
+    <SplitPageLayout.Footer divider="none">
+      <Placeholder label="Footer" height={64} />
+    </SplitPageLayout.Footer>
+  </SplitPageLayout>
+</Box>
 ```
 
 ### With pane on right
 
-```jsx
-<SplitPageLayout>
-  <SplitPageLayout.Header>
-    <Placeholder label="Header" height={64} />
-  </SplitPageLayout.Header>
-  <SplitPageLayout.Content>
-    <Placeholder label="Content" height={240} />
-  </SplitPageLayout.Content>
-  <SplitPageLayout.Pane position="end">
-    <Placeholder label="Pane" height={120} />
-  </SplitPageLayout.Pane>
-  <SplitPageLayout.Footer>
-    <Placeholder label="Footer" height={64} />
-  </SplitPageLayout.Footer>
-</SplitPageLayout>
+```jsx live drafts
+<Box sx={{height: 320, overflowY: 'auto', border: '1px solid', borderColor: 'border.default'}}>
+  <SplitPageLayout>
+    <SplitPageLayout.Header>
+      <Placeholder label="Header" height={64} />
+    </SplitPageLayout.Header>
+    <SplitPageLayout.Content>
+      <Placeholder label="Content" height={320} />
+    </SplitPageLayout.Content>
+    <SplitPageLayout.Pane position="end">
+      <Placeholder label="Pane" height={120} />
+    </SplitPageLayout.Pane>
+    <SplitPageLayout.Footer>
+      <Placeholder label="Footer" height={64} />
+    </SplitPageLayout.Footer>
+  </SplitPageLayout>
+</Box>
 ```
 
 ### Without padding
```

Comment:
Non-blocking nitpick: it could be nice to add an example where padding is changed to one of the other values. That seems like the more common use case, but i could be wrong.

Reply:
Done 👍